perllib/AngusSOAP.pm

```diff
@@ -0,0 +1,168 @@
+package AngusSOAP;
```

Comment:
I realise this matches the existing one, but perhaps we should move that and this into a subdirectory.